scripts/inspect_ckpt.py

```
@@ -0,0 +1,229 @@
+    W(f"  Per-ch mean: min={cm.min():.4f} max={cm.max():.4f} range={cm.max()-cm.min():.4f}")
+    W(f"  Per-ch std:  min={cs.min():.4f} max={cs.max():.4f} ratio={cs.max()/cs.min():.2f}x")
+    si = cs.argsort()
+    W(f"  Lowest-var:  chs={si[:5].tolist()} stds={[round(x,4) for x in cs[si[:5]].tolist()]}")
+    W(f"  Highest-var: chs={si[-5:].tolist()} stds={[round(x,4) for x in cs[si[-5:]].tolist()]}")
+    W(f"  All 64 means: {[round(x,3) for x in cm.tolist()]}")
+    W(f"  All 64 stds:  {[round(x,4) for x in cs.tolist()]}")
+    del slc, cm, cs; gc.collect()
+
+    W("\n=== DONE ===")
+
+except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)